conf/application.conf

```diff
@@ -35,6 +35,10 @@ xsd.locations.fileupload += "/api/conf/1.0/schemas/wco/declaration/FileUpload.xs
 
 xml.max-errors = 25
 
+circuitBreaker.numberOfCallsToTriggerStateChange = 5
```

Comment:
should we add CB on/off as well?

Reply:
setting a high value for e.g. numberOfCallsToTriggerStateChange will achieve the same outcome so leaving unaltered